pubspec.yaml

```diff
@@ -8,4 +8,5 @@ environment:
   sdk: ">=2.0.0 <3.0.0"
 
 dev_dependencies:
+  pedantic: 1.1.0
```

Comment:
missed this earlier - should this be `^1.1.0` to accept new versions?

Reply:
Do we want to keep this locked hard so we don't have no lints sprung on us causing a break?

Is adding a lint considered breaking?

Reply:
IMO it should be treated along the lines of `@deprecated`. If `package:pedantic` is published with new lints our travis will go red until we address it, which is fine.